installscripts/wizard/scenarios/support/jazz_jenkins.py

```diff
@@ -124,18 +129,20 @@ def get_and_add_docker_jenkins_config(jenkins_docker_path):
     """
         Launch a dockerized Jenkins server.
     """
-    os.chdir(jenkins_docker_path)
-    print("Running docker launch script")
-    subprocess.call([
-        'sg', 'docker', './launchscript.sh', '|', 'tee', '-a',
-        '../../docker_creation.out'
-    ])
-    # Get values to create the array
-    parameter_list = []
-    with open("docker_jenkins_vars") as f:
-        for line in f:
-            parameter_list.append(line.rstrip())
+    print "\nPlease provide Subnets details, since ECS Fargate will be configured with ALB"
+    jenkins_subnet = raw_input("Please enter the subnetid 1 :")
+    jenkins_subnet2 = raw_input("Please enter the subnetid 2 :")
+    encrypt_passwd = hashlib.md5()
+    encrypt_passwd.update(str(datetime.datetime.now()))
+    jenkins_passwd = encrypt_passwd.hexdigest()
+    jenkins_elb = str(ec2_metadata.public_ipv4)
+    jenkins_security_group = str(ec2_metadata.network_interfaces[ec2_metadata.mac].security_group_ids[0])
```

Comment:
We need the vpc id, security group and subnets for configuring the Application load balancer and ECS fargate. Here, the vpc id and security group are reading from the aws configured instance and we are assuming that the same VPC id and SG group will be used for configuration. Only part which we did not get is the security groups, which ALB needs at least two. So that we will ask during the installation.

Reply:
Updated.